openlabcmd/openlabcmd/cli.py

```diff
@@ -54,6 +55,28 @@ def _add_check_cmd(self, parser):
         cmd_check.add_argument('--recover', action='store_true',
                                help='Enable the auto recover mode.')
 
+    def _add_repo_cmd(self, parser):
+        # openlab repo list
+        cmd_repo = parser.add_parser(
+            'repo',
+            help='The repos which enable the OpenLab.')
+        cmd_repo_list_sub = cmd_repo.add_subparsers(title='repo',dest='repo')
+        cmd_repo_list = cmd_repo_list_sub.add_parser(
+            'list', help='List the repos which enbale the OpenLab app.')
+
+        cmd_repo_list.set_defaults(func=self.repo_list)
+        cmd_repo_list.add_argument('--server', default='github.com',
+                                   help="Specify base server url. Default is "
+                                        "github.com")
+        cmd_repo_list.add_argument(
+            '--app_id', default='7102',
+            help="Specify the github APP ID, Default is 7102 (allinone: 7102,"
+                 " OpenLab: 6778).")
+        cmd_repo_list.add_argument(
+            '--app_key', default='/var/lib/zuul/openlab-app-key.pem',
+            help='Specify the app key file path. Default is '
```

Comment:
how about --app-** for keeping the same format like others

Reply:
++, use "-" instead.

Reply:
done